### PR TITLE
Issue#1108 movie cards and banners

### PR DESCRIPTION
**Pull request by ioaNikas**
- [x] InVision style for movie cards and carousels
- [x] Mat-menu moved outside movie-card (specific to some apps)
- [x] Added wishlist functions to cards and banners
- [x] Replace poster with banner in marketplace home

### CODE REVIEW

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html [hunk 1]
```diff
@@ -19,18 +19,34 @@ <h4>{{ section.subline}} </h4>
         </aside>
         <mat-carousel test-id="mat-carousel1" proportion="50" [hideArrows]="true" [hideIndicators]="true">
           <!-- We only want to use the first image in the promotinalElements.images array-->
-          <ng-container *ngFor="let movie of section.movies">
-            <mat-carousel-slide #matCarouselSlide [image]="movie.main.poster" [hideOverlay]="true">
+            <mat-carousel-slide *ngFor="let movie of section.movies" [image]="movie.main.poster" [hideOverlay]="true">
               <div fxLayout="column" fxLayoutAlign="end start" class="black-bg-opacity-30 overlay">
+                <ng-container *ngIf="(isAddedToWishlist(movie.id) | async); else alreadyAdded">
+                  <button mat-icon-button (click)="addToWishlist(movie)">
+                    <mat-icon svgIcon="heart_filled"></mat-icon>
```
Comment (Maxstgt) [inline apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: I use the heart filled icon when the movie is already added. I guess you doing it here as well, just want to make sure.

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html [hunk 2]
```diff
@@ -19,18 +19,34 @@ <h4>{{ section.subline}} </h4>
...
+                  </button>
+                </ng-container>
+                <ng-template #alreadyAdded>
+                  <button mat-icon-button (click)="addToWishlist(movie)">
+                    <mat-icon svgIcon="heart"></mat-icon>
+                  </button>
+                </ng-template>
                 <footer fxLayout="column" fxLayoutAlign="center start">
+                  <b *ngIf="movie.main.status as status" class="status-text">{{ status.toUpperCase() }}</b>
```
Comment (Maxstgt) [inline apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: Vincent wanted me to use for the movie view the movie status which is inside of the static model...so maybe you want to use the slug pipe here, so you don't have the slug in the frontend`{{ movie.main.status | translateSlug: 'MOVIE_STATUS'}}`

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html [hunk 3]
```diff
@@ -19,18 +19,34 @@ <h4>{{ section.subline}} </h4>
...
                   <h2>{{ movie.main.title.original }}</h2>
-                  <span *ngFor="let director of movie.main.directors">{{ director.firstName }}
-                    {{ director.lastName }}</span>
-                  <span><b>Production year</b> {{ movie.main.productionYear }}</span>
+                  <span *ngFor="let director of movie.main.directors">{{ director.firstName }} {{ director.lastName }}</span>
+                  <b>
+                    <span *ngFor="let country of movie.main.originCountries | slice:0:2; let isLast=last">
+                      {{ country | translateSlug: 'TERRITORIES'}}{{isLast ? ' ' : ', '}}
```
Comment (Maxstgt) [inline apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: sexy

File: apps/catalog/marketplace/marketplace/src/app/movie/search/search.component.ts [hunk 4]
```diff
@@ -375,6 +379,20 @@ export class MarketplaceSearchComponent implements OnInit, OnDestroy {
     }
   }
 
+  public isAddedToWishlist(movieId: string): Observable<boolean> {
```
Comment (Maxstgt) [inline apps/catalog/marketplace/marketplace/src/app/movie/search/search.component.ts]: maybe put it inside of the service? cause you using it here and in the home.component

File: libs/ui/src/lib/movie-card/movie-card.component.html [hunk 5]
```diff
@@ -7,27 +7,10 @@
...
-        <mat-icon color="primary">delete_outline</mat-icon>
-        <span>Delete</span>
-      </button>
-    </mat-menu>
-    <mat-menu #subMenu="matMenu">
-      <a mat-menu-item [routerLink]="['../', movie.id]">Explorer</a>
-    </mat-menu>
+    <!-- Movie Status Chip -->
+    <mat-chip selected disableRipple class="movie-chip">
+      <span class="table-size-text">{{ movie.main.status.toUpperCase() }}</span>
```
Comment (Maxstgt) [inline libs/ui/src/lib/movie-card/movie-card.component.html]: use the slug pipe here as well

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html (hunk 1 above)
Comment (ioaNikas) [reply apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: Indeed, it's a massive copy/paste from your code and behaviour with minor changes :)

File: apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html (hunk 2 above)
Comment (ioaNikas) [reply apps/catalog/marketplace/marketplace/src/app/movie/home/home.component.html]: My mistake, I didn't know you created a pipe to transform the slug into label, this is cool

**Pull request merged**